lib/counter_culture/counter.rb

```diff
@@ -176,6 +201,12 @@ def first_level_relation_foreign_key
       relation_reflect(first_relation).foreign_key
     end
 
+    def polymorphic?
+      relation_reflect(relation).polymorphic?.tap do |is_polymorphic|
+        raise "Polymorphic associations only supported with one level" unless (relation.is_a?(Symbol) || relation.length == 1) if is_polymorphic
```

Comment:
@magnusvk I've started on this, but realize there's more than one way of having a polymorphic association in a multi-level relation (beginning, middle and end). I'd like to work on support for **multi-level** polymorphic as a separate PR (maybe after the other two refactoring PRs) -- would you be happy to merge this one as is?